### PR TITLE
Add About page

### PR DESCRIPTION
**Pull request by notklaatu**
The current site doesn't really explain what Silverblue is, at least not in an easy-to-find place and in an easy quotable "elevator pitch".

I've added an About page to help people understand what the project is.

### CODE REVIEW

File: pages/about/.backups/index.html~ [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!-- See structure.html for the template names used here and their position in the layout. -->
```
Comment (travier) [inline pages/about/.backups/index.html~:1]: You probably did not intend to add this one here.

**Review by travier — COMMENTED** [review]
I'm OK with that but isn't all of this already part of the main page in the documentation?

File: pages/about/.backups/index.html~ (hunk 1 above)
Comment (notklaatu) [reply pages/about/.backups/index.html~]: Oops, sorry. Removed in a (squashed) commit.

**Comment by notklaatu** [comment]
I find it difficult to quickly find a "blurb" about Silverblue on this site. Now that you point out that introductory information is in the Documentation link, I do see that it's there. 

In practise, though, I think it's better to have a quick explanation of what the project is about on the main landing page of the project. I did spend five minutes trying to locate an "about" block (ironically to prove a friend wrong about the site having no explanation of the project), but it never occurred to me to click the Documentation link because (I thought to myself) *I wasn't looking for information on how to install it or how to use it*. I just wanted to see a few sentences explaining what it was. 

Right or wrong, I think many users are most likely to click on an About page for a quick and easy explanation of the project. I'm no SEO expert, but I'm willing to bet it probably better to have that blurb on the project subdomain rather than to "hide" it on docs.fedoraproject.org

File: pages/about/index.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!-- See structure.html for the template names used here and their position in the layout. -->
+{{define "title"}}Team Silverblue &mdash; About{{end}}
+
+{{define "body"}}
+
+<article>
+<h1>What is Silverblue?</h1>
+<p>
+  Silverblue is a new generation of the desktop operating system formerly known as Atomic Workstation.
```
Comment (travier) [inline pages/about/index.html:9]: Can you update that with the intro text from https://docs.fedoraproject.org/en-US/fedora-silverblue/ ?

**Comment by tpopela** [comment]
Thank you @notklaatu !

**Pull request merged**